components/file_upload/file_upload.jsx

```diff
@@ -47,7 +47,7 @@ const holders = defineMessages({
 
 const OVERLAY_TIMEOUT = 500;
 
-class FileUpload extends React.PureComponent {
+class FileUpload extends PureComponent {
```

Comment:
Any particular motivation for this change?

Reply:
I just thought it's our convention, but when I checked again (after your comment) I realized it's not :(